core/tests/performance_framework/selenium_driver.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Selenium + browsermobproxy
+
+This file contains selenium + browsermobproxy to fetch performance data.
+"""
+
+import urlparse
+import os
+
+CURR_DIR = os.path.abspath(os.getcwd())
```

Comment:
Yes, a cleaner way to do this would be much better. I was searching for something like this in the code base. I found it here: core/tests/gae_suite.py:35.